util/portmuxusr.c

```diff
@@ -160,18 +159,17 @@ static int remaining_timeoutms(int startms, int timeoutms)
 }
 
 static int portmux_cmd(const char *cmd, const char *app, const char *service,
-                       const char *instance, char *post)
+                       const char *instance, const char *post)
 {
     char name[strlen(app) + strlen(service) + strlen(instance) + 3 +
               MAX_DBNAME_LENGTH];
     char res[32];
     SBUF2 *ss;
-    int rc, fd, port;
```

Comment:
I really don't see any problems with the C90 style. Let's not change it.

Reply:
Tried to make this behave similarly in many of the functions below. There is no gain here, I agree.